### PR TITLE
Confused some peopls

### PR DESCRIPTION
**Pull request by Feel65**
I hate the transgirl tag get rid of it please

### CODE REVIEW

**Comment by KTibow** [comment]
Learn json first

File: docs/assets/tags.json [hunk 1]
```diff
@@ -21,8 +21,8 @@
     ],
     "HellCastleSupportCustomTag4": ["&8[&aS&2t&eo&6n&9k&8]", "&8[&2$$$&8]"],
     "HellCastleSupportCustomTagFeel": [
-      "&8[&bTr&dan&fs&dgi&brl&8]",
-      "&8[&bF&de&fl&8]"
+      "&c[AMON&fGU&cS] super sus ,
```
Comment (KTibow) [inline docs/assets/tags.json:24]:
```suggestion
      "&c[AMON&fGU&cS] super sus",
```

**Comment by Feel65** [comment]
no it was intentional

**Comment by Feel65** [comment]
is it good now

File: docs/assets/tags.json [hunk 2]
```diff
@@ -21,8 +21,8 @@
     ],
     "HellCastleSupportCustomTag4": ["&8[&aS&2t&eo&6n&9k&8]", "&8[&2$$$&8]"],
     "HellCastleSupportCustomTagFeel": [
-      "&8[&bTr&dan&fs&dgi&brl&8]",
-      "&8[&bF&de&fl&8]"
+      "&c[AMON&fGU&cS] super sus",
+      "&cSussy "
```
Comment (KTibow) [inline docs/assets/tags.json:25]:
```suggestion
      "&cSussy"
```

**Comment by KTibow** [comment]
Your fault if your tag looks weird after this

**Comment by Feel65** [comment]
patch notes: added jank

**Pull request merged**